src/about/about.tsx

```diff
@@ -0,0 +1,6 @@
+// watch: native intellisense and file-peek for aliases from jsconfig.json and with none-js files doesn't work: https://github.com/microsoft/TypeScript/issues/29334
+
+function About() {
```

Comment:
There is no need to use casual function declaration. Instead, you can use arrow function and make your component look shorter:
```suggestion
const About = () => <div> About </div>

export default About;
```

Do it, please, for the **rest** of your functional components